core/templates/dev/head/domain/exploration/InteractionObjectFactory.js

```diff
@@ -27,7 +28,7 @@ oppia.factory('InteractionObjectFactory', [
     this.confirmedUnclassifiedAnswers = confirmedUnclassifiedAnswers;
     this.customizationArgs = customizationArgs;
     this.defaultOutcome = defaultOutcome;
-    this.fallbacks = fallbacks;
+    this.fallbacks = generateFallbacksFromBackend(fallbacks);
```

Comment:
We need a better name to distinguish the backend fallbacks. Maybe fallbacksBackendList or similar?

(Btw, please double-check to make sure that any args which represent the backend representation are renamed correctly, otherwise we're going to end up with general developer confusion.)